.github/workflows/codeql-query.yml

```diff
@@ -135,7 +135,7 @@ jobs:
           signed_auth_token: ${{ github.event.inputs.signed_auth_token }}
 
       - name: Handle workflow failed
-        if: failure() && github.event.inputs.variant_analysis_id != '' && github.run_attempt == 1
```

Comment:
I think we can also make the `variant_analysis_id` required in the `query/action.yml` file, but this can perhaps be done in a different PR since I believe that the `signed_auth_token` should also be required now.

Reply:
Yeah I agree there's some signed auth token changes we can make too. Since we're happy with this as it is I'll merge this now and open a new PR to make both of the fields required.

Reply:
Followup PR: https://github.com/github/codeql-variant-analysis-action/pull/802